coralproject/talk
# [CORL-1188] Add media to comment data downloads

## What does this PR do?

This updates the admin and user CSV downloads to include a "Media" column that will specify the media type and the URL linked to the comment.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Create some GIF, Tweet, and YouTube media comments
- Go to My Profile > My Comments  and download your comments
- Open the download invite email
- Download your comments to a zipped CSV
- Unpack the zip and load up the CSV
- See a `Media` column with media URL entries
